website/docs/networking/vpc-lattice/traditionalabtesting.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Traditional A/B testing"
+sidebar_position: 15
+---
+
+
+We have modified the `checkout` microservice adding a feature that provides an additional shipping option. In this section, we want to test this new feature performing A/B testing. 
+
+Let's deploy a new version of the `checkout` microservice in a new namespace (`checkoutv2`) using `kustomize`.
+
+```bash
+$ kubectl apply -k workspace/modules/networking/vpc-lattice/abtesting/
```

Comment:
Instead of creating a new namespace for v2 and patching the config, can we simply just create another deployment pointing to the new image? We can use a different name (to avoid conflict), but the same labels. This allows both versions of the pod to be served by the same service. This is rudimentary traffic routing, with a 50/50 chance of hitting v1 vs v2. This also removes the need to patch the config.

Optionally, you can then delete the v1 deployment afterwards. This allows cleanup to be much easier as well.